tests/test_models.py

```diff
@@ -73,7 +73,7 @@ def test_example_replace_this(self):
         self.assertIsNotNone(resource.id)
         found = Order.all()
         self.assertEqual(len(found), 1)
-        data = Order.find(resource.id)
-        self.assertEqual(data.name, resource.name)
+        # data = Order.find(resource.id)
+        # self.assertEqual(data.name, resource.name)
```

Comment:
Maybe we can remove this test case altogether?

Reply:
sure